samples/sensor/soc_voltage/boards/nucleo_g071rb.overlay

```diff
@@ -0,0 +1,12 @@
+/ {
```

Comment:
While you're at, let's move all these to `boards/`

Reply:
OK, I'll add the `vbat` node to the appropriate SoCs to compliment this as well.